provision/acc_provision/templates/connectivity-checker-cr.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: nettools.debug/v1alpha1
+kind: ConnectivityChecker
+metadata:
+ name: test-conn-check
+ namespace: nettools
+spec:
+ reachability_test_enable:
+  periodic_sync:
+   enable_updates: {{ config.connectivity_checker.reachability_test_enable.periodic_sync.enable_updates }}
+   interval: {{ config.connectivity_checker.reachability_test_enable.periodic_sync.interval }}
+  external_url:
+   url: {{ config.connectivity_checker.reachability_test_enable.external_url.url }}
+  proxy:

```

Comment:
We need to first check if the cko_proxy_config field is present in the input file. {% if config.cko_proxy_config %}
Only then set the proxies.

Reply:
addressed